spongehome.go

```diff
@@ -26,23 +26,65 @@
 package main
 
 import (
-    "github.com/SpongePowered/SpongeHome/controllers"
-    "github.com/go-macaron/pongo2"
-    "gopkg.in/macaron.v1"
+	"fmt"
+	"github.com/SpongePowered/SpongeHome/controllers"
+	"github.com/go-macaron/pongo2"
+	"github.com/sethvargo/go-fastly"
+	"gopkg.in/macaron.v1"
+	"os"
+	"time"
 )
 
+func clearfastly() {
+	//Clear Fastly cache
+	if os.Getenv("FASTLY_KEY") != "" {
+		time.Sleep(10 * time.Second) //let the http server startup
+		fmt.Println("starting fastly client")
+		client, err := fastly.NewClient(os.Getenv("FASTLY_KEY"))
+		if err != nil {
+			fmt.Println(err)
+		}
+		services, err := client.ListServices(&fastly.ListServicesInput{})
+		FASTLY_NAME := ""
+		if os.Getenv("SPONGE_ENV") == "prod" {
+			FASTLY_NAME = "www.spongepowered.org"
+		} else if os.Getenv("SPONGE_ENV") == "staging" {
+			FASTLY_NAME = "www-staging.spongepowered.org"
+		}
+		for _, svc := range services {
+			if svc.Name == FASTLY_NAME {
+				_, err := client.PurgeAll(&fastly.PurgeAllInput{
+					Service: svc.ID,
+					Soft:    false})
```

Comment:
Is gofmt actually OK with you leaving `})` on the end like that? I'd prefer if it was on its own line.